docs/self-hosted/operations/15_observability.md

```diff
@@ -0,0 +1,12 @@
+---
+id: observability
+title: Observability
+---
+
+## Prometheus
+
+Ory technology exposes an endpoint for snapshot Prometheus data. This endpoint is usually found at the administrative port for Ory
```

Comment:
> Ory technology exposes an endpoint for snapshot Prometheus data.

Unclear. What do you mean? That there is an endpoint where you can get snapshots of data collected by Prometheus?